tests/contrib/dbapi/test_unit.py

```diff
@@ -0,0 +1,205 @@
+import mock
+import pytest
+
+from ddtrace import Pin, Span
+from ddtrace.contrib.dbapi import TracedCursor, TracedConnection
+from ddtrace.ext import AppTypes, sql
+from tests.test_tracer import get_dummy_tracer
+
+
+class TestTracedCursor(object):
```

Comment:
This would require some effort with `pytest` and our use of fixtures in this test. Worth doing? https://docs.pytest.org/en/latest/unittest.html#mixing-fixtures

Reply:
based on my other comment below, might not be needed anyways.

in the future, this test case will probably inherit `from tests.base import BaseTracerTestCase` which configures a `self.tracer` and handles the teardown and etc for us.

also, it should "just work" ™️

Reply:
should just work in the sense, if you change this from `(object):` to `(unittest.TestCase):` should still run fine... _should_